Classes/MapEditor/MapEditor.cs

```diff
@@ -355,6 +356,58 @@ public class MapEditor {
         }
         AddNotes(notes);
     }
+
+public void QuantizeSelection() {
+        if (currentMapDifficulty == null) {
+            return;
+        }
+
+        // default value of 1/4 tact if no bpm and grid change is made
+        int defaultGridDivision = 4;
```

Comment:
have not found the value. any suggestion?

Reply:
`Helper.DoubleParseInvariant((string)mapEditor.GetMapValue("_editorGridSpacing", RagnarockMapDifficulties.Current, custom: true));`